packages/millicast-sdk/src/types/index.d.ts

```diff
@@ -1424,6 +1424,12 @@ declare module '@millicast/sdk' {
 
     /** The audio media ID of RTCRtpTransceiver */
     audioMid?: string;
+
+    /** The average target latency, Can be set to 0, 
+     * enabling zero-buffering mode (which is not recommended as it affects video playback smoothness). 
+     * The default value is 100 ms, except when PlayReady or Widevine L1 on Windows are used - those 
```

Comment:
non blocking, but because we don't support PlayReady, we can remove it from here

Reply:
I just copied this from castlab SDK's integration guid, and there is also `play ready` url from director subscribe API's response